bin/dump-postgres.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DATE=$(date +"%Y-%m-%d-%H-%M")
+
+DUMPDIR=/home/noiapp/dump
```

Comment:
consider to create `dump` dir if missing to avoid failure.

Reply:
done